wiki/Music_theory/Syncopation/en.md

```diff
@@ -0,0 +1,7 @@
+---
+stub: true
+---
+
+# Syncopation
+
+**Syncopation** is the accenting of a note which would usually not be accented. Syncopation can also be mapped in somewhat by placing notes off-beat i.e. red or blue timeline ticks. The [time signature](/wiki/Music_theory/Time_signature) of a piece of music gives an indication of a regular [rhythm](/wiki/Music_theory/Rhythm) pattern of strong and weak beats. A syncopated rhythm goes against this expectation by putting the accent on weak beats, thereby creating counterflow.
```

Comment:
this may need an example (even an image?) to better convey the topic

Reply:
ill mark as todo, as i dont really have an awful lot of time at the moment, a lot of stuff going on irl